docs/features/configuration.adoc

```diff
@@ -0,0 +1,41 @@
+Templates for Issue and Test Management links can be set by system properties:
+
+- `allure.link.issue.pattern`
+
+- `allure.link.tms.pattern`
+
+Default `allure-results` directory location can also be overridden by setting +
+`allure.results.directory` system property
+
+[source, xml, linenums]
+.pom.xml
+----
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>2.20</version>
+            <configuration>
+                ...
+                <systemPropertyVariables>
+                    <allure.results.directory>${project.build.directory}/allure-results</allure.results.directory>
+                    <allure.link.issue.pattern>https://example.org/browse/{}</allure.link.issue.pattern>
+                    <allure.link.tms.pattern>https://example.org/browse/{}</allure.link.tms.pattern>
+                </systemPropertyVariables>
+            </configuration>
+            ...
+        </plugin>
+    </plugins>
+</build>
+----
+
+Or by placing `allure.properties` file to resources folder: `src/test/resources`
```

Comment:
I'd like to have `allure.properties` file mentioned first as the main (*recommended*) way to configure Allure